assemblies-valid/AssembliesValid.cs

```diff
@@ -149,6 +149,8 @@ static Machine GetCurrentMachine(Architecture arch)
 #if NET6_0_OR_GREATER
                 case Architecture.S390x:
                     return Machine.Unknown;
+		case Architecture.Ppc64le;
```

Comment:
Do we need to guard this with `#if NET7_0_OR_GREATER`? Otherwise .NET 6 will try and compile this code too. Is `Ppc64le` present in .NET 6?

Reply:
Ppc64le is not present in .NET6 , so yet we need to guard this with NET7_0_OR_GREATER